src/server/rest/v1/service/TagService.ts

```diff
@@ -248,6 +248,20 @@ export default class TagService {
     // Check User
     UtilsService.assertObjectExists(action, user, `User ID '${filteredRequest.userID}' does not exist`,
       MODULE_NAME, 'handleUpdateTag', req.user);
+    if (tag.visualID !== filteredRequest.visualID) {
+      // Check visualID uniqueness
+      const tagVisualID = await TagStorage.getTagByVisualID(req.user.tenantID, filteredRequest.visualID);
```

Comment:
If the visual ID is not unique, the DB will throw an error. You just want to make the error more explicit to the admin user? 
Since it's not a hot code path, the extra calls to the DB are fine by me.